### PR TITLE
WIP:feat(nodejs): nodejs with dind

### PR DESCRIPTION
**Pull request by caosm**
- dockerfile of nodejs with docker-in-docker

### CODE REVIEW

File: nodejs/8.9/alpine/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,49 @@
+FROM cargo.caicloudprivatetest.com/caicloud/node:8.9-alpine
```
Comment (bbbmj) [inline nodejs/8.9/alpine/Dockerfile]: base 镜像就是`node:8.9-alpine`吧，改一下

File: nodejs/8.9/alpine/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+		--directory /usr/local/bin/ \
+	; \
+	rm docker.tgz; \
+	\
+	apk del .fetch-deps; \
+	\
+	dockerd -v; \
+	docker -v
+
+CMD [ "node" ]
```
Comment (bbbmj) [inline nodejs/8.9/alpine/Dockerfile]: 加个空行

**Comment by bbbmj** [comment]
目标镜像是啥？ 改下目录结构

**Comment by caosm** [comment]
don't need

**Pull request closed without merging**